corehq/form_processor/abstract_models.py

```diff
@@ -155,6 +155,65 @@ def get_subcases(self):
     def get_case_property(self, property):
         raise NotImplementedError
 
+    def get_json(self, lite=False):
```

Comment:
Good point.
After reviewing this comment I realized it was actually overkill to move this whole function to `AbstractCommCareCase`, so I'm going to move it back to `CommCareCase`, and just use tastypie's serialization workflow to grab the bits it needs (properties and indices) while leaving `get_json()` unchanged: https://github.com/dimagi/commcare-hq/pull/10976/commits/e84e725d6241b167966d2262c3e2d96004a6fbde